int16[] ids
geometry_msgs/Polygon[] polygons
